packages/thumbprint-react/components/Tooltip/index.tsx

```diff
@@ -60,9 +60,9 @@ export interface TooltipProps {
      */
     children: (args: ChildrenPropTypes) => JSX.Element;
     /**
-     * Plain text that will appear within the tooltip. Links and formatted content are not allowed.
+     * ReactNode that will appear within the tooltip. Links and formatted content are allowed.
      */
-    text: string;
+    text: React.ReactNode | Text;
```

Comment:
I chatted with @egoens and we'd rather have an API where the CTA is a separate prop. For example:

```
cta: {
  href: string;
  text: string;
} | {
  onClick: string;
  text: string;
}
```

This is better for both design consistency and accessibility since we can guarantee that the colors will be accessible.

Reply:
I see! I am wondering what is `cta` needs to be place between somewhere in side text, not at the end. How does this work in that case?🤔

Reply:
We'd re-design in that case to move the CTA to the end rather than in-between the text. Feel free to reach out to @egoens on Slack for design help!

Reply:
Sure! Updated code to support cta!👍